Adding meeting notes for Editors meeting 43 (#247)
cardano-foundation/CIPs: BiweeklyMeetings/2022-04-12.md

**M. Àngels Jover**-: Yes. This one was agreed last week to merge it, but I have added it to triage as I haven't seen it merged yet.

**Matthias Benkort**-:  Yeah. Right, right, right. I wanted to go through it, which I actually did yesterday. Yeah, you just need a number. That's why we haven't still merged it. But it's tentatively CIP 40, so it's just a matter now of moving the file to the right folder, putting CIP 40 as a number, and getting it merged, indeed. That we discussed, but we also haven't proved it explicitly. Yeah, I will approve and reduce conditions of those terms. I'm not sure, Robert, if you had time to go or to look at this one, but the status quo of what we discussed last week was that this has been made and written in combination with the ledger team, and it's actually already being adopted in the ledger as an implementation and should be part of the Vasil Hard Fork coming in June, so it's a bit fast tracked in that sense.

**Robert Phair**-: : Yeah. That's all I got from it is that it seems to be in progress anyway, and I certainly don't have any disagreement with it, the way it's being treated.

> **Matthias Benkort**-:  Yeah, it was more like an adult conversation that got documented as a CIP, but by the time the CIP arrived, the decision had already been made with the ledger team about the direction to take. It's more an informational CIP in that sense.

Reviewer: ```suggestion
**Matthias Benkort**-:  Yeah, it was more like an ad-hoc conversation that got documented as a CIP, but by the time the CIP arrived, the decision had already been made with the ledger team about the direction to take. It's more an informational CIP in that sense.
```

**Robert Phair**-: : It's fait accompli, yeah.

**Matthias Benkort**-:  And therefore, we could also maybe address the status already, because at least as proposed. Yeah, can you confirm we expect this to be included in the Vasil Hard Fork? I would say we could merge it as proposed and move it to active as soon as it's observable after the Hard Fork. Let's document that under pull request review change status to propose.

**Matthias Benkort**-:  Okay, sorry. That's commented directly on the ticket.

**M. Àngels Jover**-: Here, we have the update. Perfect. We are missing one approval right here, right now, to merge it.

> **Matthias Benkort**-:  Yeah, but want to merge it also right away, as I said. We still need a few bullet points to be tackled, so either Sebastian can do it or any of the editor can tackle that later on. We should be able to push on the branch, at least.

Reviewer: ```suggestion
**Matthias Benkort**-:  Yeah, but we don't want to merge it also right away, as I said. We still need a few bullet points to be tackled, so either Sebastian can do it or any of the editor can tackle that later on. We should be able to push on the branch, at least.
```

#### PR163
 [PR163 - CIP-37? | Dynamic Saturation based on Pledge](https://github.com/cardano-foundation/CIPs/pull/163)

**M. Àngels Jover**-: Okay. I just added a few things to the agenda that were... This one, for example, 163 was on hold, and you comment that Matthias said was sitting with the engineering team. I was wondering if you had any update on this.

> **Matthias Benkort**-:  Yeah. Unfortunately, no. This one is a bit the same as the curve pledge benefits CIP seven, or another CIP, which I don't remember the... Yeah, 24th non-centralized ranking which are affecting the protocol and the rewards mechanism of the protocol or the way pools get rewards and get organized. And therefore, this is affecting the game theory behind the scene, and that's CIPs for which we wanted to have inputs from the researchers. From what I got so far is that the research team has no bandwidth to provide an actual detailed answer, but that they are not quite in favor of these new CIPs because they put some... Well, they reduced the Nakamoto coefficient or they put some security issues, but they have been quite vague so far on the reasons and the hashes behind, so I'm really trying to get a formal answer or some more detailed answer on those matters. But yeah, so no substantial updates from my side, unfortunately.

Reviewer: ```suggestion
**Matthias Benkort**-:  Yeah. Unfortunately, no. This one is a bit the same as the curve pledge benefits CIP seven, or another CIP, which I don't remember the... Yeah, 24th non-centralized ranking which are affecting the protocol and the rewards mechanism of the protocol or the way pools get rewards and get organized. And therefore, this is affecting the game theory behind the scene, and that's CIPs for which we wanted to have inputs from the researchers. From what I got so far is that the research team has no bandwidth to provide an actual detailed answer, but that they are not quite in favor of these new CIPs because they put some... Well, they reduced the Nakamoto coefficient or they put some security issues, but they have been quite vague so far on the reasons and the rationales behind, so I'm really trying to get a formal answer or some more detailed answer on those matters. But yeah, so no substantial updates from my side, unfortunately.
```
